### ISSUE TITLE
training problem 

### ISSUE THREAD

**Issue by may-lin12** [OpenNMT/OpenNMT-py#1169]
Hello there, I tried to train the model following the instruction on OpenNMT github on a GPU machine, but got problems. After prepossing the data, I enter the command:

python train.py -data data/data -save_model model_20190110 -word_vec_size 500 -encoder_type brnn -train_steps 10000 -enc_layers 1 -dec_layers 1 -rnn_size 300 -world_size 1 -gpu_ranks 1 

I also tried transformer model. the command:

python train.py -data data/data -save_model model_20190110 -layers 4 -rnn_size 128 -word_vec_size 128 -max_grad_norm 0 -optim adam -encoder_type transformer -decoder_type transformer -position_encoding -dropout 0.1 -param_init 0 -warmup_steps 8000 -learning_rate 2 -decay_method noam -label_smoothing 0.1 -adam_beta2 0.998 -batch_size 1024 -batch_type tokens -normalization tokens -max_generator_batches 2 -train_steps 10000 -accum_count 4 -copy_attn -param_init_glorot -world_size 1 -gpu_ranks 1 -save_checkpoint_steps 10000

my dataset is LCSTS(chinese summarization dataset). When I was training, both model (RNN and transformer) loading dataset again and again. I don't know what's going on?
![image](https://user-images.githubusercontent.com/17610219/50985593-a7920b80-153f-11e9-9380-2111c328bf00.png)

I tried for a long time, but never succeed. Who can help me solve the problems please? Thank you very much!!

**Comment by vince62s** [comment]
read the doc please (multi gpu part)
https://github.com/OpenNMT/OpenNMT-py/blob/master/docs/source/FAQ.md
